### DISCUSSION TITLE
Users activity tag

### DISCUSSION THREAD

**Discussion by TomasAdvance** [frangoteam/FUXA#433] [Ideas]
Hi Frango team,
Interested new function will be user activity log tag. That function could be very help full during investigation if something is happed, in control process. It will be nice to see, who and what change.

What do you thing to implemented Users activity log record?
Technicaly could be an system tag where sould be generated json string with following data sets :
1) time stamp
2) loged User
3) type of action
4) old value
5) new value

**Comment by TomasAdvance** [comment]
Hi Fuxa team,
Do you thinking about that feature in fuxa?
T

**Reply by unocelli** [reply]
Hi, we have planed in one of the next versions
